tests/ui/components/filters/daterangecomponent.js

```diff
@@ -0,0 +1,376 @@
+import DOM from '../../../../src/ui/dom/dom';
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DateRangeFilterComponent from '../../../../src/ui/components/filters/daterangefiltercomponent';
+import FilterNodeFactory from '../../../../src/core/filters/filternodefactory';
+import Filter from '../../../../src/core/models/filter';
+
+describe('date range filter component', () => {
+  DOM.setup(document, new DOMParser());
+
+  let COMPONENT_MANAGER, defaultConfig, setFilterNode;
+  const metadataFormatters = {
+    greaterThan: min => `After ${min}`,
+    greaterThanEqual: min => `${min} or later`,
+    lessThan: max => `Before ${max}`,
+    lessThanEqual: max => `${max} and earlier`,
+    exclusiveRange: (min, max) => `${min} - ${max}`,
+    inclusiveRange: (min, max) => `Between ${min} and ${max}`
+  };
+
+  beforeEach(() => {
+    const bodyEl = DOM.query('body');
+    DOM.empty(bodyEl);
+    DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+    setFilterNode = jest.fn();
+
+    const mockCore = {
+      setFilterNode: setFilterNode,
+      filterRegistry: {
+        setFilterNode: setFilterNode
+      }
+    };
+
+    COMPONENT_MANAGER = mockManager(
+      mockCore,
+      DateRangeFilterComponent.defaultTemplateName()
+    );
+
+    defaultConfig = {
+      container: '#test-component'
+    };
+  });
+
+  it('renders correctly for default values', () => {
```

Comment:
For this and the other Filter components, do we have full test coverage for the different display string scenarios?

Reply:
Added coverage for exclusive range for date range filter, geolocation, and filtersearch
I believe it should be 100% or very close now!